README.md

```diff
@@ -15,19 +15,29 @@ Then install the application by going to cftool directory and typing:
 
 `./build.sh`
 
-Configuration file (config.yaml) will be copied to `/etc/.Appliscale/cftool`
+Configuration file (config.yaml) will be copied to your home directory under the `/.Appliscale/cftool/config.yaml` path.
 
 The application should be compiled to `cftool` binary file to the `bin` directory in your GO workspace.
 
 To validate your template online, with AWS API, just type:
 
-`cftool -mode=validate -file=[path to your template] -region=[region, e.g. eu-central-1]`
+`cftool -mode=validate -file=[path to your template]`
 
 To validate template offline (well, almost offline - AWS CloudFormation Resource Specification still needs be downloaded) use validate_offline mode:
 
-`cftool -mode=validate_offline -file=[path to your template] -region=[region]`
+`cftool -mode=validate_offline -file=[path to your template]`
 
 To convert your template from JSON to YAML and form YAML to JSON type:
 
 `cftool -mode=convert -file=[path to your template] -output=[path to place, where you want to save converted file] 
 -format=[JSON or YAML]`
+
+
+## Configuration file
```

Comment:
AFAIK you can have one line before heading and one line after heading and result will be the same.